packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -108,6 +108,12 @@ export const ChatMessageComponentAsRichTextEditBox = (
     [iconClassName]
   );
 
+  /* @conditional-compile-remove(attachment-upload) */
+  const hasMultipleAttachments = useMemo(() => {
```

Comment:
Should we move this to utils files (as it's shared between edit box and rich text edit box)?

Reply:
done